apps/tsilhqotin-language-hub/src/Components/Widgets/Navbar/dropdown.tsx

```diff
@@ -1,36 +1,32 @@
+import { Link } from 'react-router-dom';
+
 export function Dropdown() {
     return (
         <div>
-            <a className="menu__item" href="/About">
+            <Link className="menu__item" to="/About" reloadDocument={true}>
```

Comment:
Consider a config. Definitely try to generate this more dynamically.

Reply:
Think of what will be easier as far as maintenance. What is the easiest way to update the list of menu items?